src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/PackageModelFactory.cs

```diff
@@ -69,6 +91,26 @@ public PackageModel Create(PackageSearchMetadataContextInfo metadata, ContractIt
             }
         }
 
+        private PackageModel Create(VersionInfoContextInfo version)
+        {
+            EmbeddedResourcesCapability embeddedResources = new EmbeddedResourcesCapability(_packageFileService, version.PackageSearchMetadata.Identity!, version.PackageSearchMetadata.ReadmeUrl);
+
+            if (version.PackageSearchMetadata.PackagePath != null)
```

Comment:
Inverting the logic will make the `else` condition easier to understand.  Right now, the `else` condition is "not not null."